src/main/java/codeu/controller/AdminServlet.java

```diff
@@ -6,18 +6,117 @@
+    request.setAttribute("numMessages", numMessages());
+    request.setAttribute("mostActiveUser", mostActiveUser());
+    request.setAttribute("newestUser", newestUser());
+    request.setAttribute("wordiestUser", wordiestUser());
     request.getRequestDispatcher("/WEB-INF/view/admin.jsp").forward(request, response);
   }
+
+  /** Returns number of users in database */
+  private int numUsers() {
+    return this.userStore.getUsers().size();
+  }
+
+  /** Returns number of conversations in database */
+  private int numConvos() {
+    return this.conversationStore.getConversations().size();
+  }
+
+  /** Returns number of messages in database */
+  private int numMessages() {
+    return this.messageStore.getMessages().size();
+  }
+
+  /** Calculated by user with most messsages sent and most conversations started */
+  private String mostActiveUser() {
+    // Mapping User's UUID to the frequency of their contributions
+    HashMap<UUID, Integer> frequencyMap = new HashMap<UUID, Integer>();
+    String mostActiveUserName = "No messages/conversations posted";
+    List<Conversation> conversations = this.conversationStore.getConversations();
+    List<Message> messages = this.messageStore.getMessages();
+
+    for (Conversation conversation : conversations) {
+      UUID conversationOwner = conversation.getOwnerId();
+      frequencyMap.merge(conversationOwner, 1, Integer::sum);
+    }
+
+    for (Message message : messages) {
+      UUID messageOwner = message.getAuthorId();
+      frequencyMap.merge(messageOwner, 1, Integer::sum);
+    }
+
+    try {
+      // Finding UUID with max value (contributions)
+      UUID mostActiveUserUUID = frequencyMap.entrySet().stream().max((entry1, entry2) -> entry1.getValue() > entry2.getValue() ? 1 : -1).get().getKey();
+      mostActiveUserName = this.userStore.getUser(mostActiveUserUUID).getName();
+    } catch (Exception e) {
+      System.out.println(e.getMessage());
+    }
+    return mostActiveUserName;
+  }
+
+  /** Returns newest member by creation date */
+  private String newestUser() {
+    String newestUser = "No users added";
+    Instant newestUserCreationTime = Instant.MIN;
+    List<User> users = this.userStore.getUsers();
+
+    for (User user : users) {
+      if (user.getCreationTime().isAfter(newestUserCreationTime)) {
+        newestUserCreationTime = user.getCreationTime();
+        newestUser = user.getName();
+      }
+    }
+
+    return newestUser;
+  }
+
+  /** Returns User with max individual words sent */
+  private String wordiestUser() {
+    // Mapping User's UUID to the frequency of their words sent
+    HashMap<UUID, Integer> frequencyMap = new HashMap<UUID, Integer>();
+    String wordiestUser = "No users added";
+    List<Message> messages = this.messageStore.getMessages();
+
+    for (Message message: messages) {
+      UUID messageOwner = message.getAuthorId();
+      int wordsInMessage = message.getContent().split("\\w+").length;
```

Comment:
It's better to add getNumWords method to Message and put this logic there. Rule of thumb: if your piece of code uses information from just one object and it is easy to come up with a function name for what you are doing, that piece of code should be a method of that object. <- This is probably too confusing though, so let's see how that applies here.

Consider this line above.
1. Takes message from one object, splits it into words, counts the number of those words. Just one object is used -- Check.
1.5 Nothing else is using that data in surrounding code -- Check. A bonus one :)
2. Is there a simple descriptive name we can give to function, that describes what we are doing? getNumberOfWords or getNumWords (if you don't want to be too verbose) sound like good candidates. -- Check

So, the code needs to be moved into the object.